### PR TITLE
add docs for new labels

### PR DESCRIPTION
**Pull request by FFMMM**
Docs update for https://github.com/hashicorp/consul/pull/12727

Signed-off-by: FFMMM <FFMMM@users.noreply.github.com>

### CODE REVIEW

File: website/content/docs/agent/telemetry.mdx [hunk 1]
```diff
@@ -551,6 +552,23 @@ Historically, `internal` RPC operation metrics were accounted under the same met
...
+| Label Name                            | Description                                                                    | Possible values                                            |
+| ------------------------------------- | ------------------------------------------------------------------------------ | ---------------------------------------------------------- |
+| `blocking`                            | Whether the read request passed in a `MinQueryIndex`.                          | `true` if a MinQueryIndex was passed, `false` otherwise.   |
+| `target_datacenter`                   | The target datacenter for the read request.                                    | The string value of the target datacenter for the request. |
+| `locality`                            | Gives an indication of whether the RPC request is local or has been forwarded. | `local` or `forwarded`.                                    |
+
+##### Read Request Label Explanations
+
+We determine the values for the `locality` label by looking at the current server datacenter and the request target datacenter. If they match, we call that `local` RPC request.
+Otherwise, we call that RPC request `forwarded`.
```
Comment (msiege2) [inline website/content/docs/agent/telemetry.mdx:570]: Let's rephrase this to avoid first person language.  Suggestion:

"The values for the `locality` label are determined by comparing the current server datacenter and the request target datacenter.  When those values are equal, the RPC request locality is `local`, otherwise locality is `forwarded`.

Alternately, remove this section and just add to the table above in the Possible values column:

`local` if current server data center is the same as target_datacenter, otherwise `forwarded`.

**Comment by msiege2** [comment]
I made a small comment.  Small enough that you don't need to re-review after changing, so approved.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/635551.

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 59c25cf89123b1f0a849a4d9e1841f82a4727f7d onto `stable-website` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/635551)

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 59c25cf89123b1f0a849a4d9e1841f82a4727f7d onto `release/1.12.x` succeeded!
